envycontrol.py

```diff
@@ -38,7 +38,7 @@
 alias nvidia_current_modeset off
 '''
 
-UDEV_INTEGRATED_PATH = '/lib/udev/rules.d/50-remove-nvidia.rules'
+UDEV_INTEGRATED_PATH = '/etc/udev/rules.d/50-remove-nvidia.rules'
```

Comment:
@PilotGuy772 could you please add a new variable instead? As systems running older versions may have these files on their previous location the cleanup routine will not remove them

Reply:
Another solution would be to manually add the paths to the cleanup routine that way we won't need a new variable